jacklib/api.py

```diff
@@ -213,12 +213,15 @@ def _d(s, encoding=ENCODING):
 JackBBTFrameOffset = 0x40
 JackAudioVideoRatio = 0x80
 JackVideoFrameOffset = 0x100
+# JACK2 1.9.19+:
+JackTickDouble = 0x200
 JACK_POSITION_MASK = (
     JackPositionBBT
     | JackPositionTimecode
     | JackBBTFrameOffset
     | JackAudioVideoRatio
     | JackVideoFrameOffset
+    | JackTickDouble
```

Comment:
this doesnt match jack2 headers. `JACK_POSITION_MASK` in jack2 only has `JackPositionBBT|JackPositionTimecode` set. but that seems to be a jack2 issue.
then jack1/shared is missing the new `JackTickDouble` hmmm :/

not a problem for you, just noticed this inconsistency in jack headers